docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -62,10 +74,21 @@ When right clicking an element, you are able to edit, delete, duplicate, and mod
 
 {{< figure src="/static/img/docs/canvas-panel/canvas-context-menu-9-2-0.png" max-width="750px" caption="Canvas element context menu" >}}
 
-## Canvas Options
+## Canvas options
 
 ### Inline editing
 
 The inline editing toggle enables you to lock or unlock the canvas panel. When turned off the canvas panel becomes “locked”, freezing elements in place and preventing unintended modifications.
 
 {{< video-embed src="/static/img/docs/canvas-panel/canvas-inline-editing-toggle-9-2-0.mp4" max-width="750px" caption="Inline editing toggle demo" >}}
+
+### Data links
+
+Canvas supports [data links](https://grafana.com/docs/grafana/latest/panels-visualizations/configure-data-links/). Once you've added a data link to the panel, you can display it by following these steps:
+
+1. Set an element to be tied to a field value.
+1. Turn off the inline editing toggle.
```

Comment:
Are these numbers supposed to increment?

Reply:
I believe they auto increment - [see this comment](https://github.com/grafana/grafana/pull/62923#discussion_r1100398712)

Reply:
Nice!